internal/levee.go

```diff
@@ -118,3 +123,16 @@ func getArgumentPropagator(c *config.Config, call *ssa.Call) ssa.Node {
 
 	return nil
 }
+
+func isFieldPropagator(c *ssa.Call, fieldPropagators map[types.Object]bool) bool {
+	for o, _ := range fieldPropagators {
```

Comment:
Aside: if you only need the keys from a map, you can do `for key := range myMap`.

Reply:
Thank you, I keep forgetting that :blush:

Reply:
[Tone: playful ribbing of a `vim` user]
<cough> an IDE would highlight it <cough>.

Reply:
Haha, `vim-go` actually nits me on it too, but it's not super visible :/